package.json

```diff
@@ -3,43 +3,41 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@types/node": "^16.11.11",
-    "@types/react": "^17.0.37",
+    "@types/node": "^17.0.8",
+    "@types/react": "^17.0.38",
     "@types/react-dom": "^17.0.11",
     "classnames": "^2.3.1",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "react-scripts": "4.0.3",
-    "sass": "^1.44.0",
+    "react-scripts": "5.0.0",
+    "sass": "^1.48.0",
     "serve": "^13.0.2",
-    "typescript": "^4.5.4"
+    "typescript": "^4.5.4",
+    "web-vitals": "^2.1.3"
```

Comment:
🗒️ this is a new package, create-react-app adds it (and the required boilerplate) by default. With it you can report metrics like "first input delay", "time to first byte", etc from real users. I would use it on my next project, so I left it in. But maybe this warrants a separate PR? More info here: [GooggleChrome/web-vitals](https://github.com/GoogleChrome/web-vitals)